gen_install.sh

```diff
@@ -10,17 +10,34 @@ sudo apt-get -qq update -y
 
 if(($(cat /etc/os-release | grep VERSION_ID|grep -o '".*"' | sed 's/"//g' | cut -c1-2 )==16));then
     ROS_VERSION="kinetic"
-    else
-    if(($(cat /etc/os-release | grep VERSION_ID|grep -o '".*"' | sed 's/"//g' | cut -c1-2 )==18)); then
+fi    
+if(($(cat /etc/os-release | grep VERSION_ID|grep -o '".*"' | sed 's/"//g' | cut -c1-2 )==18)); then
     ROS_VERSION="melodic"
 fi
+if(($(cat /etc/os-release | grep VERSION_ID|grep -o '".*"' | sed 's/"//g' | cut -c1-2 )==20)); then
+    ROS_VERSION="noetic"
 fi
 echo "installing for ros version: ${ROS_VERSION}"
 sudo apt-get install -qq -y ros-${ROS_VERSION}-desktop-full
 source /opt/ros/${ROS_VERSION}/setup.bash
 ## install rosmon, it would be weird for this to be in one of the packages:
 #sudo apt-get install ros-${ROS_VERSION}-rosmon
+if(($ROS_VERSION="kinetic"||$ROS_VERSION="melodic")); then
```

Comment:
I think you need square brackets here for string comparisons (https://stackoverflow.com/a/12765593/5274985)

Reply:
I looked at the same stack overflow post earlier yesterday, I think I might have forgotten it when I made the changes again, will make the change